File: BusinessLogic/Models/Persistency/Action.cs
namespace kRetro.BusinessLogic.Models.Persistency
{
    public enum ActionStatus{
        New=0,
        Open=1,
        Done=3
    }
    public class Action
    {
        public int Id {get;set;}
        public string Description {get;set;}
        public string InChargeTo {get;set;}
        public string WhoChecks {get;set;}
        public CardBase Card {get;set;}
        public ActionStatus Status {get;set;}
    }
}

File: ClientApp/models/persistency/action.ts
import { CardBase } from "./cardBase";

export enum ActionStatus {
    New = 0,
    Open = 1,
    Done = 3
}

export class Action {
    public id: number | undefined;
    public description: string | undefined;
    public inChargeTo: string | undefined;
    public whoChecks: string | undefined;
    public card: CardBase | undefined;
    public status: ActionStatus | undefined;
}